src/styles/index.scss

```diff
@@ -0,0 +1,25 @@
+@import url('https://fonts.googleapis.com/css2?family=Marcellus&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Figtree:ital,wght@0,300..900;1,300..900&display=swap');
```

Comment:
Why does one of them have `:ital,wght@0,300..900;1,300..900` and the other doesn't? Is is what Google Fonts gives you by default?

Reply:
yes, token from Google fonts